android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/notifications/OnVehicleDataTests.java

```diff
@@ -320,6 +320,9 @@ public void testJson() {
             bodyInformationObj.put(BodyInformation.KEY_PARK_BRAKE_ACTIVE, VehicleDataHelper.BODY_INFORMATION_PARK_BRAKE);
             bodyInformationObj.put(BodyInformation.KEY_IGNITION_STABLE_STATUS, VehicleDataHelper.BODY_INFORMATION_IGNITION_STATUS);
             bodyInformationObj.put(BodyInformation.KEY_IGNITION_STATUS, VehicleDataHelper.BODY_INFORMATION_IGNITION_STABLE_STATUS);
+            bodyInformationObj.put(BodyInformation.KEY_ROOF_STATUSES, VehicleDataHelper.ROOF_STATUES);
```

Comment:
@kostyaBoss I think you can revert these changes, as vehicle data does not need to be changed for this PR. The corresponding PR for the JavaScript suite (https://github.com/smartdevicelink/sdl_javascript_suite/pull/354/files) did not make changes to the vehicle data.